Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-start.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
+
 <body>
-  <script src="https://statics.teams.cdn.office.net/sdk/v1.5.2/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
+  <script src="https://statics.teams.cdn.office.net/sdk/v1.5.2/js/MicrosoftTeams.min.js"
+    crossorigin="anonymous"></script>
   <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal.min.js" crossorigin="anonymous"></script>
   <script type="text/javascript">
     microsoftTeams.initialize();
     microsoftTeams.getContext(function (msTeamsContext) {
 
       // ADAL.js configuration
       let config = {
-        clientId: "fa792ad0-1f96-4807-b652-929e8efbaf95",
+        clientId: "da0770c2-f800-4ab4-870e-8ffae5d39f6c",
```

Comment:
Ditto for this one. Not sure if this is on purpose or if the developer is expected to replace it with their own clientId.